content/docs/user-guide/basic-concepts/data-artifact.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 'Data Artifact'
+match: ['data artifact', 'data artifacts']
+---
+
+Any data file or directory, as well as intermediate or final result that is
+tracked by DVC, for example by using 'dvc add'. See
```

Comment:
My bad Roger - it's not even single quotes, but backtick ` ... line inline-code block in Markdown.

Reply:
Ohhh, I see! I'll fix that up tomorrow while checking over all the changes again myself.